pkg/util/rate_limiter.go

```diff
@@ -0,0 +1,14 @@
+package util
+
+import (
+	"flag"
+	"time"
+)
+
+func init() {
+	flag.DurationVar(&BaseDelay, "rate-limiter-base-delay", time.Millisecond*5, "The base delay for rate limiter. Defaults 5ms")
+	flag.DurationVar(&MaxDelay, "rate-limiter-max-delay", time.Minute, "The max delay for rate limiter. Defaults 1min")
```

Comment:
`time.Minute` might break the default `1000s`

Reply:
done